src/sentry/static/sentry/app/views/dashboardsV2/controls.tsx

```diff
@@ -132,17 +137,9 @@ class Controls extends React.Component<Props> {
             value={currentOption}
             onChange={({value}: {value: DashboardListItem}) => {
               const {organization} = this.props;
-
-              if (value.type === 'prebuilt') {
-                browserHistory.push({
-                  pathname: `/organizations/${organization.slug}/dashboards/`,
-                  query: {},
-                });
-                return;
-              }
-
               browserHistory.push({
                 pathname: `/organizations/${organization.slug}/dashboards/${value.id}/`,
+                // TODO(mark) should this retain global selection?
```

Comment:
is this a TODO you meant to leave in?

Reply:
Yeah, I want to talk with Jeremy and come to a decision on this.